### PR TITLE
Feature: Adding CreateMetadataAccount

### PR DESCRIPTION
**Pull request by olegfomenko**
Adding create metadata account program instruction

### CODE REVIEW

**Comment by yihau** [comment]
Hi, thank for your contribution. Would you add a test for it?
also I don't return error in instruction function. I think the serialize error should and never happen.

**Comment by olegfomenko** [comment]
> Hi, thank for your contribution. Would you add a test for it? also I don't return error in instruction function. IMHO the serialize error shouldn't and never happen.

Hi, ok, i'll fix it

**Comment by olegfomenko** [comment]
> Hi, thank for your contribution. Would you add a test for it? also I don't return error in instruction function. IMHO the serialize error shouldn't and never happen.

Done

File: program/metaplex/tokenmeta/instruction.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+			},
+			{
+				PubKey:     mintAuthority,
+				IsSigner:   true,
+				IsWritable: false,
+			},
+			{
+				PubKey:     payer,
+				IsSigner:   true,
+				IsWritable: false,
```
Comment (yihau) [inline program/metaplex/tokenmeta/instruction.go:51]: according to https://github.com/metaplex-foundation/metaplex/blob/master/rust/token-metadata/program/src/instruction.rs#L270
I think it should be
```diff
PubKey:     payer,
IsSigner:   true,
- IsWritable: false,
+ IsWritable: true,
```

Comment (olegfomenko) [reply program/metaplex/tokenmeta/instruction.go]: Very strange thing - it have worked for me on testnet. I've fixed it, it still works

**Comment by yihau** [comment]
Thank you!

**Pull request merged**